prisma/schema.prisma

```diff
@@ -7,7 +7,7 @@ generator client {
 }
 
 generator json {
-  provider = "prisma-json-types-generator"
+  provider = "node ./dist/generator.js"
```

Comment:
I was previously using `npm link` and `npm install -g`, but this option should be preferred.

Reply:
Nice, I see! We'll leave it in, then!